### ISSUE TITLE
ListPlayers command  crashes application every few hours

### ISSUE THREAD

**Issue by komatooo** [WeLoveSquad/squad-server-info-discord-bot#28]
Every few hours application crashes with following exception:

```javascript
Executing [ListPlayers] command
node:internal/buffer:88
  throw new ERR_OUT_OF_RANGE(type || 'offset',
  ^

RangeError [ERR_OUT_OF_RANGE]: The value of "offset" is out of range. It must be >= 0 and <= 8130. Received 8133
    at boundsError (node:internal/buffer:88:9)
    at Buffer.readInt32LE (node:internal/buffer:392:5)
    at Rcon.readPacket (file:///home/node/app/build/rcon/rcon.js:168:27)
    at Rcon.decodeData (file:///home/node/app/build/rcon/rcon.js:151:33)
    at Socket.<anonymous> (file:///home/node/app/build/rcon/rcon.js:50:18)
    at Socket.emit (node:events:514:28)
    at addChunk (node:internal/streams/readable:545:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:495:3)
    at Readable.push (node:internal/streams/readable:375:5)
    at TCP.onStreamRead (node:internal/stream_base_commons:190:23) {
  code: 'ERR_OUT_OF_RANGE'
}
```

Despite the fact its restarting automatically, still requires manual initialization from discord.

**Comment by Mikerosoft25** [comment]
Hey, could you also attach the log lines prior to this error?
I tried to reproduce this error but could not do it. Our bot is running for the last 15 days without ever running into this error.
Is your Squad Server running with any mods?

> Despite the fact its restarting automatically, still requires manual initialization from discord.

Do you mean, that you have to use the `/init` and `init-player-info` command again after the bot restarts? This should not happen since the bot should save it's previously initialized channels and reuse those after restarting, if you use the docker-compose in this repo. If that is not the case for you, then I have to look into that as well.

**Comment by Mikerosoft25** [comment]
@komatooo I think i have found the code segment that could cause your error when some edge case occurred. I think I have fixed that issue in v3.0.1. Could you please use that version and check, if the bot does not crash anymore? If the bot still crashes, please open another issue, or reopen this one, if you can.
